### ISSUE TITLE
nlp.sentences (or just nlp) produces wrong dot at the end

### ISSUE THREAD

**Issue by q-kirill** [spencermountain/compromise#559]
I'm getting such strange response when there is a whitespace in between sentence and dot.

`nlp('Once or twice .').text() // "Once or twice . ."`

The same when:

`nlp('Once or twice .').sentences().text() // "Once or twice . ."`

Is it a bug or such behaviour is correct for some reason?

**Comment by zzj0402** [comment]
Don't seem like an issue here.
```ts
test('Dot test', function (t) {
  var doc = nlp('Once or twice .');
  t.equal(doc.text(), 'Once or twice .');
  t.equal(doc.sentences().text(), 'Once or twice .');
  t.end();
});
# Dot test
ok 13 should be equal
ok 14 should be equal
```

**Comment by q-kirill** [comment]
Thanks @zzj0402 , the problem seems resolved in the latest 11.13.0 version.
